airflow/cli/cli_parser.py

```diff
@@ -540,9 +540,18 @@ def string_lower_type(val):
 )
 ARG_IGNORE_DEPENDS_ON_PAST = Arg(
     ("-I", "--ignore-depends-on-past"),
-    help="Ignore depends_on_past dependencies (but respect upstream dependencies)",
+    help="Deprecated -- use `--depends-on-past ignore` instead. "
+    "Ignore depends_on_past dependencies (but respect upstream dependencies)",
     action="store_true",
 )
```

Comment:
We should emit a RemoveInAirflow3Warning somewhere when this is used.

Reply:
done